arch/arm64/configs/bcm2711_defconfig

```diff
@@ -1134,6 +1136,7 @@ CONFIG_VIDEO_BCM2835=m
 CONFIG_MAILBOX=y
 CONFIG_BCM2835_MBOX=y
 # CONFIG_IOMMU_SUPPORT is not set
+CONFIG_BCM2835_POWER=y
```

Comment:
Instead of adding such a fundamental driver to a config, we can make it default for the architekture.

https://github.com/raspberrypi/linux/commit/f494caa480f7089ff9bba16dd865e190cee7e545#diff-f3d0c8e9a196caa504677d8d31a131c3

Reply:
Done

Reply:
Please preserve the authorship of commits and cherry-pick it as an indiviual patch. Squashing all into one makes everything hard to maintain.